static/js/compose_closed_ui.js

```diff
@@ -24,7 +24,8 @@ export function update_reply_recipient_label(message) {
             recipient_label = message.display_reply_to;
         }
     }
-    $(".compose_reply_button_recipient_label").text(recipient_label);
+    const preprend_text = $t({defaultMessage: "Message"});
+    set_reply_button_label(`${preprend_text} ${recipient_label}`);
```

Comment:
This doesn't work i18n-wise for languages with different word order than English.  We want this:
```
+    set_reply_button_label($t({defaultMessage: "Message {recipient_label}"},
+                              {recipient_label}));
```

https://zulip.readthedocs.io/en/latest/translating/internationalization.html